frontend/src/pages/projects/screens/spawner/environmentVariables/AlertWarningText.tsx

```diff
@@ -0,0 +1,98 @@
+import { Alert, List, ListItem, Stack, StackItem } from '@patternfly/react-core';
+import React from 'react';
+
+type AlertWarningTextProps = {
+  deletedConfigMap: string[];
+  deletedSecret: string[];
```

Comment:
nit: Wondering if this component needs the context of whether the resources are "deleted" or not. Maybe just calling these `configMap` and `secret` would suffice.

Reply:
Also, should these be pluralized since they're lists of items?